testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapptabs/DemoAppMainActivity.kt

```diff
@@ -147,4 +153,21 @@ class DemoAppMainActivity : BaseActivity() {
             fragment
         } else null
     }
+
+    override fun onStart() {
+        super.onStart()
+        miniAppIdAndViewMap.forEach {
+            with(AppSettings.instance.universalBridgeState) {
```

Comment:
I'm not sure, if there is any spec to send the Json message to MiniApp from anywhere from HostApp or only from the QA settings page 🤔 cc: @rleojoseph @munir-rakuten @GondiTilak 

#### If only from the QA settings page:
Then - we don't need to cache anything like `UniversalBridgeState` or even the message content if we can send the message directly from the `QASettingsActivity` after pressing the button. Inside, `QASettingsActivity` we can access `miniAppIdAndViewMap` which was defined in `DemoAppMainActivity`.

Then, we don't need to do `sendJsonToMiniApp` at `onStart` here...

Reply:
I also think no need for any shared preference to cache the data, just send the `miniAppView` list to settings page and send the event the time user press the send  button.

Reply:
I don't get the point here.. Considering `DemoAppMainActivity` and `QaSettingsActivity` are having different lifecycles, would you explain a bit about why we don't need these...? 🤔

Reply:
aaah... Bringing the HashMap. Alright

Reply:
let's try using static then

Reply:
I think, we can declare it as a global variable e.g. 

```
val miniAppIdAndViewMap = ...
class DemoAppMainActivity : BaseActivity() { ... }
```